modules/ROOT/partials/configuration/format_noneditable_selector.adoc

```diff
@@ -0,0 +1,22 @@
+[[format_noneditable_selector]]
+== `+format_noneditable_selector+`
+
+This option allows a non-editable (`+contenteditable="false"+`) element that matches the selector specified to be wrapped with a format.
+
+Formats that can wrap include block formats that have the `+wrapper+` parameter set to `+true+` and inline formats. Selector formats are not supported.
+
+In addition to a non-editable element matching the selector, any non-editable element that has the `+data-mce-cef-wrappable+` attribute can be wrapped with a format.
+
+NOTE: The non-editable element must not have any editable descendants in order for it to be wrappable.
+
+*Type:* `+String+`
+
+=== Example: Using `+format_noneditable_selector+`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  format_noneditable_selector: 'span,p.special'
```

Comment:
```suggestion
  format_noneditable_selector: 'span, p.special'
```
should there be space here?

Reply:
Normally we don't put a space when defining a selector so would say it isn't needed